app/lib/wca_events.rb

```diff
@@ -90,19 +90,19 @@
     formats: [ "Mean of 3", "Best of x" ]
   },
   {
-    name: "4x4 blindfolded",
+    name: "4x4 Blindfolded",
     handle: "4b",
     wca_handle: "444bf",
     formats: [ "Best of x" ]
   },
   {
-    name: "5x5 blindfolded",
+    name: "5x5 Blindfolded",
     handle: "5b",
     wca_handle: "555bf",
     formats: [ "Best of x" ]
   },
   {
-    name: "3x3 multi blind",
+    name: "3x3 Multi-Blind",
```

Comment:
Looks like this should be "3x3x3 Multi-Blind"? Same with 4x4 (4x4x4) and 5x5 (5x5x5)